google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugProcessStateController.java

```diff
@@ -41,6 +43,8 @@
   private static final int INITIAL_DELAY_MS = 2000;
   private static final Logger LOG = Logger.getInstance(CloudDebugProcessStateController.class);
   private static final int PERIOD_MS = 500;
+  public static final String CLIENT_VERSION = "google.com\\/intellij\\/v" + ServiceManager
```

Comment:
This isn't actually a constant.

I'd prefer public final String clientVersion;  and then initialize it in the constructor.